server/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+const config = require(__dirname + '/../config/config.js')[env];
+const db = {};
+
+let sequelize;
+if (config.use_env_variable) {
+  sequelize = new Sequelize(process.env[config.use_env_variable], config);
+} else {
+  sequelize = new Sequelize(config.database, config.username, config.password, config);
+}
+
+fs
```

Comment:
The syntax in this file is kinda funny--did you copy it from somewhere? We can clean it up:
```js
const db = fs
  .readdirSync(__dirname)
  .filter(file => file.indexOf('.') !== 0 && file !== basename && file.slice(-3) === '.js')
  .reduce((acc, file) => {
    const model = sequelize.import(path.join(__dirname, file));
    return {
      ...acc,
      [model.name]: model
    };
  }, { sequelize, Sequelize });
```

Reply:
Also, generated by ```sequelize-cli```. I agree and like your version. Will update.